flink-end-to-end-tests/test-scripts/docker-hadoop-secure-cluster/Dockerfile

```diff
@@ -128,7 +128,7 @@ RUN set -x \
     && echo "Port 2122" >> /etc/ssh/sshd_config
 
 # Hdfs ports
-EXPOSE 50470 9000 50010 50020 50070 50075 50090 50475 50091 8020
+EXPOSE 9000 50010 50020 50070 50075 50090 50475 50091 8020
```

Comment:
Why do you remove `50470` here? Is it because we switch to random ports for the namenode's HTTPS port? If yes: Why don't we also remove `50070` as well? It's the namenode's HTTP port which also became random ([source](https://ambari.apache.org/1.2.3/installing-hadoop-using-ambari/content/reference_chap2_1.html)).